tests/CommandlineTest.php

```diff
@@ -12,22 +12,37 @@ class CommandlineTest extends OpenApiTestCase
 {
     use UsesExamples;
 
+    private function getCommandToExecute(string $cmd, bool $appendDevNull = true): string
```

Comment:
I think it would simplify things even more if we optionally passed in the redirect and make 2nd paramter `?string $devNullRedir = null`.

Then we can call it like 
```php
   $this->getCommandToExecute('xxx', '2>');
```

and the function would do:
```php
   if ($devNullRedir) {
       $cmd .= " $devNullRedir $devNull";
   }
```

Reply:
Done